<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportGraphicsView</class>
 <widget class="QWidget" name="ReportGraphicsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_zoomIn">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_zoomOut">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_highlightNegative">
         <property name="text">
          <string>Negative markieren</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_showMarkings">
         <property name="text">
          <string>Beschriftung anzeigen</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider_minDP">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_minDP"/>
       </item>
       <item>
        <widget class="QSlider" name="slider_maxDP">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_maxDP"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider_maxDP</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_maxDP</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_minDP</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_minDP</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_maxDP</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_maxDP</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_minDP</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_minDP</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
